app/views/admin/shared/_logo_upload.html.erb

```diff
@@ -1,11 +1,14 @@
 <div class="crayons-field mt-6 align-left" data-controller="logo-upload">
   <%= f.label :logo, class: "crayons-field__label" do %>
     Logo
-    <p id="logo-subtitle" class="crayons-field__description">Your logo will display in the upper left hand corner of your Forem. Upload a PNG or JPEG file.</p>
   <% end %>
+  <p id="logo-subtitle" class="crayons-field__description">Your logo will display in the upper left hand corner of your Forem. Upload a PNG or JPEG file.</p>
   <div class="flex flex-1 gap-4">
-    <%= f.file_field :logo, accept: allowed_types.to_s, aria_describedby: "logo-subtitle", data: { "max-file-size-mb": max_file_size.to_s, action: "change->logo-upload#previewLogo" }, aria: { describedby: "logo-subtitle" } %>
-    <div data-logo-upload-target="previewLogo">
+    <%= f.file_field :logo, accept: allowed_types.to_s, aria_describedby: "logo-subtitle", data: { "max-file-size-mb": max_file_size.to_s, action: "change->logo-upload#previewLogo" } %>
+    <div data-logo-upload-target="previewLogo" id="logo-upload-preview">
+      <% if logo.present? %>
+        <img class="site-logo__img" src="<%= logo %>" alt="<%= community_name %>">
```

Comment:
(suggestion / not blocking) - It's not a big thing, but maybe the `alt` for this should be something like "preview of uploaded logo" or something? I'm not sure what the best alt text for it is, but I think community_name isn't super helpful

Reply:
Yeah, good catch. I'll reword this.